hbase-server/src/main/java/org/apache/hadoop/hbase/procedure2/store/wal/ProcedureStoreTracker.java

```diff
@@ -15,7 +15,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.hadoop.hbase.procedure2.store;
+package org.apache.hadoop.hbase.procedure2.store.wal;
```

Comment:
You are moving the old store under wal subpackage?

Reply:
It is only used in WALProcedureStore so move it there. The plan is to remove the whole package in the future.

Reply:
Good